cpp/include/raft/util/integer_utils.hpp

```diff
@@ -24,6 +24,7 @@
  */
 
 #include <limits>
+#include <raft/core/resources.hpp>
```

Comment:
What do you need the `core/resources` for? Maybe include `raft/core/detail/macros.hpp` directly?

Reply:
Fair enough, I will change it. I was not sure about including a file from `detail` directory.

Reply:
Done.